engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -90,7 +90,7 @@ private static Path findInstallPath() {
         // Use the current directory as a fallback.
         Path currentDirectory = Paths.get("").toAbsolutePath();
         installationSearchPaths.add(currentDirectory);
-        LOGGER.info("PathManager: Working directory is " + currentDirectory);
+        LOGGER.atInfo().addArgument(currentDirectory).log("PathManager: Working directory is {}");
```

Comment:
I don't think we need fluent API here, the following should suffice:
```suggestion
        LOGGER.info("PathManager: Working directory is {}", currentDirectory);
```